unitary/examples/quantum_rpg/final_state_preparation/hadamard_hills.py

```diff
@@ -40,10 +41,241 @@
         description=(
             "You have reached the northern bank of the river and are at the edge\n"
             "of the Hadamard Hills, which mark the absolute southern border of the\n"
-            "quantum realm."
+            "quantum realm. A trail to the north leads up the hills."
         ),
         encounters=[],
         items=[CONSTRUCTION_SIGN],
-        exits={Direction.SOUTH: "classical12"},
+        exits={Direction.SOUTH: "classical12", Direction.NORTH: "hadamard2"},
+    ),
+    Location(
+        label="hadamard2",
+        title="Bottom of Superposition Trail",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  There seems to be two ways up the slope that you can go up."
+        ),
+        encounters=[],
+        exits={
+            Direction.SOUTH: "hadamard1",
+            Direction.EAST: "hadamard3_0",
+            Direction.WEST: "hadamard3_1",
+        },
+    ),
+    Location(
+        label="hadamard3_0",
+        title="Bottom of Superposition Slope",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  The trails north leads steeply up to the top of the hills.\n"
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard4_0", Direction.WEST: "hadamard2"},
+    ),
+    Location(
+        label="hadamard3_1",
+        title="Bottom of Superposition Slope",
+        description=(
+            "You are at the bottom of a steep slope that leads up to Hadamard Hills\n"
+            "proper.  The trails north leads steeply up to the top of the hills.\n"
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard4_1", Direction.EAST: "hadamard2"},
+    ),
+    Location(
+        label="hadamard4_0",
+        title="Superposition Slope",
+        description=(
+            "You are climbing up a steep slope.  To the north is the Hadamard hills,\n"
+            "and to the south, the trail leads down to the river separating the\n"
+            "classical realm from the Hadamard Hills."
+        ),
+        encounters=[],
+        exits={Direction.NORTH: "hadamard5", Direction.SOUTH: "hadamard3_0"},
+    ),
+    Location(
```

Comment:
Same comments on 3_0/3_1: 4_0 and 4_1 are identical. (Yep yep, superposition, I get it.)